ReactNativeClient/lib/joplin-renderer/utils.js

```diff
@@ -122,7 +122,7 @@ utils.resourceStatus = function(ResourceModel, resourceInfo) {
 	return resourceStatus;
 };
 
-utils.imageReplacement = function(ResourceModel, src, resources, resourceBaseUrl) {
+utils.resourceReplacement = function(ResourceModel, src, resources, resourceBaseUrl) {
```

Comment:
This name change was a mistake on my part, I thought I deleted the comment but I should have left a note. Please revert this change. Sorry about that.

Reply:
Doesn't it makes sense to call it resourceReplacement since it is being used to replace both image and audio resources with a src that can be used with HTML tags?

Reply:
That was my initial thought as well. But now I think the name comes from the fact that this function will replace a resource with the download image. Maybe we can come up with something more clear? `notReadyResourceReplacement`? But less wordy.

Reply:
Isn't it taking a src like `:/e67afd4316f44704bfe10db589525910` and returning something usable in html like `file://C:/Users/bryan/.config/joplindev-desktop/resources/e67afd4316f44704bfe10db589525910.mp3` ? It will only replace the resource with the download image if not ready.

Reply:
Fair point, resourceReplacement is probably fine in that case. Let's just wait for official word on that then.